### PR TITLE
ACM-2105 Translation string updates

### PR DESCRIPTION
**Pull request by Randy424**
Signed-off-by: randybrunopiverger <rbrunopi@redhat.com>
ACM-2105

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     " No status: ": " No status: ",
     " With violations: ": " With violations: ",
     " Without violations: ": " Without violations: ",
+    "({{0}} of {{0}})": "({{0}} of {{0}})",
```
Comment (Randy424) [inline frontend/public/locales/en/translation.json:5]: This appears in `frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/components/ChannelControl.js` but it seems it wasn't added to translation until I ran the script for my branch.

Comment (KevinFCormier) [reply frontend/public/locales/en/translation.json:5]: Please take note @jeswanke :) i18next-parser should be running during pre-commit, or you can run `npm run i18n` manually. But thank you for externalizing.

Comment (jeswanke) [reply frontend/public/locales/en/translation.json:5]: oops--lots of moving parts in. this one--thanks for tha catch

**Pull request merged**